src/rake_garden.py

```diff
@@ -0,0 +1,11 @@
+"""This module removes non rock and gravel from the garden."""
+
+
+def rake_garden(garden):
+    """The function removes non gravel or rock from the garden, and replaces it with gravel."""
+    list_garden = garden.split()
+    for i, c in enumerate(list_garden):
```

Comment:
Again, you're mis-using `enumerate`. `enumerate` is for when you need both the index and the value in a sequence. Here, you only need the value, so you can just do:
`for thing in list_garden:`